pages/platform/databases/cassandra_01_capabilities/guide.de-de.md

```diff
@@ -105,6 +105,13 @@ Here are the node types you can choose from:
 
 Right now, all nodes of a given cluster should be of the same type and live in the same region.
 
+#### Disk type
+
+The type of storage available may vary according to the region your cluster lives in. Thus, your cluster may be backed by e.g. *High Speed* or *High Speed Gen2* block storage.
+
+Also, the performance caracteristics of the various storage offerings may vary depending on e.g. the storage size your cluster uses: *High Speed* may offer better iops than *High Speed Gen2* for some disk sizes.
+
+Public Cloud Databases will select the most efficient disk type for your cluster depending on your cluster parameters.
```

Comment:
Add this link for customer to see in which region which storage technology is available: https://www.ovhcloud.com/en/public-cloud/regions-availability/